include/nil/blueprint/components/algebra/fields/plonk/logic_and_flag.hpp

```diff
@@ -253,8 +253,8 @@ namespace nil {
                 std::array<std::pair<std::size_t, std::size_t>, 4> wl;
 
                 int _idx;
-                for (int i = 0; i < component.rows_amount; i++) {
-                    for (int j = 0; j < witness_amount; j++) {
+                for (std::uint32_t i = 0; i < component.rows_amount; i++) {
```

Comment:
Probably needs to be std::size_t, including where rows_amount is declared, consider to change to size_t.